### DISCUSSION TITLE
(Solved) Bubble "Initialize" over and over

### DISCUSSION THREAD

**Discussion by DiabeticDan** [NightscoutFoundation/xDrip#2672] [Q&A] [answered]
I'm using Samsung z Flip3 5g. Bubble, US Libre14. OOP2, and xDrip+ ver - f8e759f-2023.02.02

I was using https://github.com/NightscoutFoundation/xDrip/pull/2621
I was getting frequent sensor issues it said (it wasn't the sensor as I could NFC fine into Libre Link). Bubble sees it, and talks to xDrip that it sees the Libre, Serial # and all are accurate. 

However no DATA was being displayed, I used to get it to start decoding and displaying by just going into System Status, and restarting the collector, or On the BT device tap the "Bluetooth Pairing option" which seemed to re-establish a connection and worked for at least half an hour... So I asked some questions on the facebook. 

Someone told me it was moved? or is now the Feb 2nd one? So I removed the old one and installed the new one. Now This is happening. I'm sorry for the Smattering of ScreenShots. 

Watching my BT Device screen in System Status now with the 2023.2.2 version my Bluetooth Device is connecting and Disconnecting every 2-5 seconds.... 

![327373068_1391541898249147_342973562608640300_n](https://user-images.githubusercontent.com/69942438/218590669-23b7f1c7-de6a-4fd5-8146-63f99d306a49.jpg)
![327802411_603453784948952_4384186000005077848_n](https://user-images.githubusercontent.com/69942438/218590671-46513035-984b-499b-9f48-f21ceaf87da6.jpg)
![328199905_1312743552626270_2607710716549016825_n](https://user-images.githubusercontent.com/69942438/218590674-4f431c03-e94b-44d3-b695-0fe3f6ddeabe.jpg)
![327676808_712254653713418_8605531151388883244_n](https://user-images.githubusercontent.com/69942438/218590676-c14ee55e-0a93-44a4-b73d-e32deea52496.jpg)
![328083228_5899000740149755_2508464798318244064_n](https://user-images.githubusercontent.com/69942438/218590678-958e54e9-5453-430f-8937-a79fc926e354.jpg)
![329236263_925133542265164_2525795253573425721_n](https://user-images.githubusercontent.com/69942438/218590679-4a2977c1-fa41-438f-b22b-6ce84e44613c.jpg)

**Comment by DiabeticDan** [comment, marked as answer]
TLDR; I believe it was my old OG Bubble battery being 50%. 

Seems the connection and disconnection every 2-5 seconds was a 50% battery issue with the Bubble? O.o I charged it to 83% and it started giving data to xDrip, and staying connected. I guess for the very time being, This is a "Solved issue". Sort of. I'm still getting this in my log.
![dasd](https://user-images.githubusercontent.com/69942438/218600593-ec2db124-d6fa-45fb-a32b-5b2a8addadfe.jpg)
